examples/src/content/jcr_root/apps/core-components-examples/components/accordion/clientlibs/amp/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[core.wcm.components.accordion.v1]"/>
```

Comment:
make this Clientlib category AMP specific, following what we did for the others (`core.wcm.components.accordion.v1.amp` I believe)